programs/sbf/benches/bpf_loader.rs

```diff
@@ -283,3 +307,29 @@ fn bench_instruction_count_tuner(_bencher: &mut Bencher) {
         );
     });
 }
+
+fn clone_regions(regions: &[MemoryRegion]) -> Vec<MemoryRegion> {
```

Comment:
Maybe this can be moved into the bpf_loader crate so that we don't need to add the rbpf dependency for this crate?

Reply:
The main package (solana-sbf-programs) already depends on rbpf and benches don't get separate dependencies